res/values/strings.xml

```diff
@@ -607,6 +607,8 @@
     <string name="pref_send_copy_to_self">Send Copy to Self</string>
     <string name="pref_auto_folder_moves">Automatic Moves to DeltaChat Folder</string>
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox</string>
+    <string name="pref_only_fetch_mvbox_title">Only Fetch from DeltaChat Folder</string>
+    <string name="pref_only_fetch_mvbox_explain">Ignore all other folders. Regardless of this setting, the Spam folder is sometimes checked for replies to known messages.</string>
```

Comment:
@r10s you wrote that the option description should mention that that requires the help of the server - like this?

```suggestion
    <string name="pref_only_fetch_mvbox_explain">This only works if your server somehow moves chat messages to the DeltaChat folder.</string>
```